code/modules/vore/eating/contaminate_vr.dm

```diff
@@ -65,28 +65,18 @@ var/list/gurgled_overlays = list(
 	if(gurgled)
 		decontaminate()
 
-/obj/structure/sink/attackby(obj/item/O as obj, mob/user as mob) //Wash the soggy item before it can interact with the sink.
+/obj/structure/sink/attackby(obj/item/O, mob/user) //Wash the soggy item before it can interact with the sink.
 	if(O.gurgled)
-		var/turf/location = user.loc
-		if(!isturf(location)) return
-
-		var/obj/item/I = O
-		if(!I || !istype(I,/obj/item)) return
-
-		to_chat(usr, "<span class='notice'>You start washing \the [I].</span>")
+		if(!istype(O))
+			return
+		to_chat(user, "<span class='notice'>You start washing [O].</span>")
```

Comment:
Backwards or unnecessary. It would runtime on `if(O.gurgled)` if it's not /obj/item because that's where `gurgled` is declared. You could do `if(istype(O) && O.gurgled)`.

Reply:
it checks istype(O), and `O` is defined as `obj/item/O`
I can change the parameter to `I` if you like

Reply:
Whoops!